packages/babel-traverse/test/inference.js

```diff
@@ -59,255 +58,202 @@ describe("inference", function() {
       const right = path.get("right");
       const strictMatch = left.baseTypeStrictlyMatches(right);
 
-      assert.ok(!strictMatch, "null should not match undefined");
+      expect(strictMatch).toBeFalsy();
     });
   });
   describe("getTypeAnnotation", function() {
     it("should infer from type cast", function() {
       const path = getPath("(x: number)")
         .get("body")[0]
         .get("expression");
-      assert.ok(
-        t.isNumberTypeAnnotation(path.getTypeAnnotation()),
-        "should be number",
-      );
+      expect(t.isNumberTypeAnnotation(path.getTypeAnnotation())).toBeTruthy();
     });
     it("should infer string from template literal", function() {
       const path = getPath("`hey`")
         .get("body")[0]
         .get("expression");
-      assert.ok(
-        t.isStringTypeAnnotation(path.getTypeAnnotation()),
-        "should be string",
-      );
+      expect(t.isStringTypeAnnotation(path.getTypeAnnotation())).toBeTruthy();
     });
     it("should infer number from +x", function() {
       const path = getPath("+x")
         .get("body")[0]
         .get("expression");
-      assert.ok(
-        t.isNumberTypeAnnotation(path.getTypeAnnotation()),
-        "should be number",
-      );
+      expect(t.isNumberTypeAnnotation(path.getTypeAnnotation())).toBeTruthy();
     });
     it("should infer T from new T", function() {
       const path = getPath("new T")
         .get("body")[0]
         .get("expression");
       const type = path.getTypeAnnotation();
-      assert.ok(
+      expect(
         t.isGenericTypeAnnotation(type) && type.id.name === "T",
-        "should be T",
-      );
+      ).toBeTruthy();
     });
     it("should infer number from ++x", function() {
       const path = getPath("++x")
         .get("body")[0]
         .get("expression");
-      assert.ok(
-        t.isNumberTypeAnnotation(path.getTypeAnnotation()),
-        "should be number",
-      );
+      expect(t.isNumberTypeAnnotation(path.getTypeAnnotation())).toBeTruthy();
```

Comment:
Instead of `expect(t.isFooBar(node)).toBeTruthy()`, can we use `expect(node.type).toBe("FooBar")`? (Also in other places)

Reply:
Maybe we should add a custom expectation for babel nodes.

`expect(node).nodeOfType("FooBar")`? https://facebook.github.io/jest/docs/en/expect.html#expectextendmatchers

Reply:
I tried doing [this](https://gist.github.com/devenbansod/e519bde4b2e58bbc74ea4ace732c8d22), but it seems Jest has some problem running with Node4 (works perfectly in Node6+).

We are facing a similar problem with Node4 in adding a new extension to Jest in #7549 too. Any help in this regard would be appreciated.